src/fiat/asset-transfers.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+parent: Fiat
+title: Asset Transfers
+---
+
+# Asset Transfers
+{:.no_toc}
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Create an Asset Transfer
+
+{% endpoint POST https://service.centraypay.com/api/asset-transfer-requests %}
+
+```sh
+curl -X POST "https://service.centraypay.com/api/asset-transfer-requests" \
+  -H "x-api-key: 1234"
+  -H "content-type: application/json" \
+  -d '{
+    assetType: "centrapay.nzd.main",
+    assetId: "YGRo6TYYSxH3js7",
```

Comment:
If asset type is "centrapay.nzd.main", should the assetId be required and set to walletId?

Given we've got `ledgerId` on /api/wallets, what's a use case which `assetType` wouldn't be ledgerId? e.g. Mad Mike Dollars

If we don't have to future proof, should we use `ledgerId` here?

Reply:
The intention is that asset id is conditionally required depending on the asset type. If the asset type is ledger-based, like "centrapay.nzd.main", then it would not be required. If it is an atomic/discreet asset like a voucher then the asset id would be required.

Reply:
Fixed in a pairing session 💖💖💖